app/assets/scss/_brief_submission.scss

```diff
@@ -52,6 +52,13 @@ ol.instruction-list-item-top li {
 }
 
 ol.steps {
+  @include bold-27;
+  padding-left: 1em; // 27px
+
+  @include media( $min-width: $site-width + (2 * (27px + $gutter) ) ) {
+    padding-left: 0;
+  }
+
```

Comment:
someone carelessly extended it, we pegged their changes on the additional class they added.